test/cppsuite/test_harness/workload_generator.h

```diff
@@ -89,7 +94,7 @@ class workload_generator : public component {
             testutil_check(session->create(session, collection_name.c_str(), DEFAULT_TABLE_SCHEMA));
             if (_enable_tracking)
                 testutil_check(
-                  _workload_tracking->save(tracking_operation::CREATE, collection_name, "", ""));
+                  _workload_tracking->save(tracking_operation::CREATE, collection_name, 0, ""));
```

Comment:
I take it the 0 here and the j+1 are our sequence numbers, for now its okay but will need to replaced with sequence numbering.